plugins/PluginBrowser/PluginBrowser.py

```diff
@@ -43,7 +44,14 @@ def __init__(self, parent = None):
 
         self._is_downloading = False
 
-        self._request_header = [b"User-Agent", str.encode("%s - %s" % (Application.getInstance().getApplicationName(), Application.getInstance().getVersion()))]
+        self._request_header = [b"User-Agent",
+                                str.encode("%s\%s (%s %s)" % (Application.getInstance().getApplicationName(),
```

Comment:
This should be `<product>/<product-version>`. The splash here is not right... I will fix it.

Reply:
Fixed in 0358b1fca

Reply:
Oh, thanks! 👍